<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>469</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>551</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="inserirParticipantesPage">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>511</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="insiraLabel">
           <property name="text">
            <string>Insira os participantes:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="nomeLineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="adicionarParticipanteButton">
             <property name="text">
              <string>Adicionar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeParticipanterButton">
             <property name="text">
              <string>Remover</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="historicoParticipantesToolButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="participantesLabel">
           <property name="text">
            <string>Participantes:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="participantesList"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="avancarButton">
           <property name="text">
            <string>Avançar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="inserirProdutosPage">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>513</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="inserirProdutosLabel">
             <property name="text">
              <string>Insira os produtos comprados</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="listaParticipantes2Label"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="adicionarProdutoButton">
               <property name="text">
                <string>Adicionar Produto</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removerProdutoButton">
               <property name="text">
                <string>Remover Produto</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QColumnView" name="produtosColumnView"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="voltarButton2">
           <property name="text">
            <string>Voltar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="avancarButton2">
           <property name="text">
            <string>Avançar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="discriminarConsumoPage">
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>513</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Participantes</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="participantesDiscriminaList"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>358</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <item>
            <widget class="QLabel" name="consumidosLabel">
             <property name="text">
              <string>Consumidos:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="consumidosList"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QPushButton" name="adicionaNaoConsumidoButton">
             <property name="text">
              <string>&gt;&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeNaoConsumidoButton">
             <property name="text">
              <string>&lt;&lt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <widget class="QLabel" name="naoConsumidosLabel">
             <property name="text">
              <string>Não consumidos:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="naoConsumidosList"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QPushButton" name="voltarButton3">
           <property name="text">
            <string>Voltar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="calcularButton">
           <property name="text">
            <string>Calcular</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="relatorioPage"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_12"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>556</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
   </widget>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
